am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/DefaultPermissionIntegrationTest.java

```diff
@@ -44,7 +46,7 @@ void shouldNotBeAbleToCreateDefaultPermissionWhenRoleDoesNotExist() {
     @Test
     void shouldAddNewEntryIntoDatabaseWhenUniqueEntry() {
         service.addRole(ROLE_NAME, RoleType.RESOURCE, SecurityClassification.PUBLIC, AccessType.ROLE_BASED);
-        service.grantDefaultPermission(createDefaultPermissionGrant(READ_PERMISSION));
+        service.grantDefaultPermission(createDefaultPermissionGrant(JsonPointer.valueOf(ATTRIBUTE), READ_PERMISSION));
```

Comment:
Can we change ATTRIBUTE to be of JsonPointer type? That should be our default type and toStirng is there if string is needed.

Reply:
Rest of changes in that file seems of similar nature, possible candidate for revert.